specs/latest/2.0/index.html

```diff
@@ -1376,6 +1375,7 @@ <h2 class="no-toc">Table of contents</h2>
           <tr><td>Uint32Array</td><td>UNSIGNED_INT_24_8</td></tr>
           <tr><td>Uint16Array</td><td>HALF_FLOAT</td></tr>
           <tr><td>Float32Array</td><td>FLOAT</td></tr>
+          <tr><td>ArrayBuffer</td><td>FLOAT_32_UNSIGNED_INT_24_8_REV</td></tr>
```

Comment:
ArrayBuffer is not compatible with the ArrayBufferView argument in the IDL. This change does not look good as is.

I am dubious that the added value of supporting uploads for this texture type is worth the perturbation to the IDL. Another overload would have to be added in order to support just this texture type -- assuming we don't want to use one of the other existing ArrayBufferView types for it (which would probably be confusing). The best fit would be either Float32Array or Uint32Array.